modules/core/src/index.js

```diff
@@ -61,3 +61,8 @@ export {default as _XVIZLoaderInterface} from './loaders/xviz-loader-interface';
 export {default as XVIZStreamLoader} from './loaders/xviz-stream-loader';
 export {default as XVIZLiveLoader} from './loaders/xviz-live-loader';
 export {default as XVIZFileLoader} from './loaders/xviz-file-loader';
+
+// Debug
+export {XvizWorkersStatus} from './debug/xviz-workers-status';
```

Comment:
We use all caps for *XVIZ*

Reply:
well, at least for exports.  We have a style guide for XVIZ to be either all upper or all lower, though I can't tell you where it's documented ;)